lib/src/Base/Algo/openturns/ClassifierImplementation.hxx

```diff
@@ -51,12 +51,24 @@ public:
   virtual UnsignedInteger classify(const NumericalPoint & inP) const;
   virtual Indices classify(const NumericalSample & inS) const;
 
+protected:
+  virtual Indices classifyParallel(const NumericalSample & inS) const;
+  virtual Indices classifySequential(const NumericalSample & inS) const;
+public:
+
   /** Grade a point as if it were associated to a class */
   virtual NumericalScalar grade(const NumericalPoint & inP,
                                 const UnsignedInteger outC) const;
   virtual NumericalPoint grade(const NumericalSample & inS,
                                const Indices & outC) const;
 
+protected:
+  virtual NumericalPoint gradeTBB(const NumericalSample & inS,
```

Comment:
I would prefer gradeParallel over gradeTBB, but this is not a big deal, +1 from me